README.md

```diff
@@ -43,14 +43,32 @@ The features that are currently working are the following:
 
 Requirements
 ------------------------------
-To *run* Melia, you need
-* .NET 4.5 (Mono 4 or above)
-* MySQL 5 compatible database
-
-To *compile* Melia, you need
-* C# 7 compiler, such as:
-  * [Visual Studio](http://www.visualstudio.com/en-us/products/visual-studio-express-vs.aspx) (2017 or later)
-  * [Monodevelop](http://monodevelop.com/) (With Mono version 5 or greater)
+
+### Running on Linux/macOS
+
+- [Mono 5](https://www.mono-project.com/download/stable/)
+- [MySQL 5](https://dev.mysql.com/downloads/mysql/5.7.html)
```

Comment:
Wouldn't it be better to stick to "MySQL X compatible"? In my eyes, the goto database nowadays is the MySQL fork MariaDB, which is mostly compatible, but not necessarily 100%. If I saw a link to MySQL, I might assume that I _need that_.

Reply:
That's true. MariaDB is usually the best solution. But there are some differences between MySQL and MariaDB that are important to be mentioned.

Reply:
Hence my previous phrasing of "MySQL 5 compatible", which, to me, suggests that a range of servers _could_ be compatible, which makes me think that a common denominator will work. That no MySQL- or MariaDB-only features will be used, because then it would require MySQL _or_ MariaDB.

Reply:
How about something like "MySQL 5 compatible database", but with two links, for MySQL or MariaDB, as examples?

Reply:
Makes sense!